tests/migrator/test_journals.py

```diff
@@ -232,12 +232,12 @@ def test_access_urls(app):
             {
                 "access_urls": [
                     {
-                        "access_restriction": "8",
+                        "access_restriction": ['RESTRICTED_PACKAGE_DEAL'],
                         "description": "some text",
                         "value": "https://url.cern.ch/journal",
                     },
                     {
-                        "access_restriction": "4",
+                        "access_restriction": [None],
```

Comment:
is it correct, the None?
I think we might miss a value from the mapping if it is taken from a real example (please check on CDS)
if it does not occur on CDS, then we should raise UnexpectedValue if something missing from the mapping, in order to catch all the unexpected behaviours

Reply:
It does not occur on CDS, I will change it to raise UnexpectedValue, thanks!